src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -288,12 +292,14 @@ public PaymentModificationResponse execute(@Nullable final Long transactionAmoun
 
     @Override
     public HostedPaymentPageFormDescriptor buildFormDescriptor(final UUID kbAccountId, final Iterable<PluginProperty> customFields, final Iterable<PluginProperty> properties, final CallContext context) throws PaymentPluginApiException {
+        killbillMetrics.markMeter("hpp");
```

Comment:
I would extract these strings in a shared place and document them, so it's easy to look-up later what they mean without reading the code.

For example:

```
// Could be an ENUM too...
public abstract class AdyenMetrics {
  // Rate of HPP requests
  public static final String HPP_METER = "hpp";
}
```